CredentialProvider.Microsoft/CredentialProvider.Microsoft.csproj

```diff
@@ -3,37 +3,35 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <RootNamespace>NuGetCredentialProvider</RootNamespace>
-    <TargetFrameworks>netcoreapp3.1;net6.0;net461</TargetFrameworks>
     <LangVersion>latest</LangVersion>
     <ApplicationIcon>helpericons.ico</ApplicationIcon>
     <Version>$(CredentialProviderVersion)</Version>
     <PackageVersion>$(CredentialProviderVersion)</PackageVersion>
     <NuspecFile>CredentialProvider.Microsoft.nuspec</NuspecFile>
     <!-- Shouldn't need to set these explicitly, there may be a bug in dotnet pack -->
     <NuspecProperties>$(NuspecProperties);Configuration=$(Configuration);Version=$(CredentialProviderVersion)$(VersionSuffix)</NuspecProperties>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <PlatformTarget>AnyCPU</PlatformTarget>
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <PlatformTarget>AnyCPU</PlatformTarget>
     <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
     <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Identity.Client" Version="4.32.1" />
-    <PackageReference Include="Microsoft.Identity.Client.Extensions.Msal" Version="2.18.5" />
+    <PackageReference Include="Microsoft.Identity.Client" Version="4.45.0"/>
+    <PackageReference Include="Microsoft.Identity.Client.NativeInterop" Version="0.8.0" Condition=" '$(TargetFramework)' == 'net461' "/>
```

Comment:
Hmm, is this required? I was under the impression that the ref to `Microsoft.Identity.Client.Broker` is sufficient.

Reply:
For whatever reason, the transitive dependency was not getting pulled in for net461.  I will double check.

Reply:
@bgavrilMS Confirmed.  When I comment this out and run `test.bat` I get:

```
[Verbose] [CredentialProvider]MSAL Log (Error): False MSAL 4.46.0.0 MSAL.Desktop 4.8 or later Windows 10 Enterprise [2022-08-05 15:14:11Z - c7aa1236-f1c7-45c8-8c03-8269c30990df] Exception type: System.DllNotFoundException

   at Microsoft.Identity.Client.NativeInterop.API.x64.MSALRUNTIME_Startup()
   at Microsoft.Identity.Client.NativeInterop.API.x64.Startup()
   at Microsoft.Identity.Client.NativeInterop.Module.AddRef()
   at Microsoft.Identity.Client.NativeInterop.Core..ctor()
   at Microsoft.Identity.Client.Broker.RuntimeBroker.<AcquireTokenSilentDefaultUserAsync>d__12.MoveNext()

Reply:
net6.0 and netcoreapp3.1 work fine 🤷‍♂️